rust/osm2lanes-web/src/main.rs

```diff
@@ -109,6 +109,15 @@ impl Component for App {
                 self.update_tags();
                 true
             }
+            Msg::CountrySet(Ok(country)) => {
+                self.state.locale.country = Some(country);
+                self.update_tags();
+                true
+            }
+            Msg::CountrySet(Err(country_err)) => {
+                self.state.message = Some(country_err.to_owned());
+                true
+            }
```

Comment:
maybe unnecessary, would only be triggered by client manipulation of the countries...